prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm.c

```diff
@@ -455,7 +462,6 @@ static inline void
 fi_ibv_rdm_tagged_release_remote_sbuff(struct fi_ibv_rdm_tagged_conn *conn,
```

Comment:
Please, remove seq_num as a parameter. Now, you are updating only head buffer. The same is actual for future enhancements as well.